configure.ac

```diff
@@ -3,7 +3,10 @@ dnl Process this file with autoconf to produce a configure script.
 dnl Include external macro definitions before the AC_INIT to also remove
 dnl comments starting with # and empty newlines from the included files.
 dnl ----------------------------------------------------------------------------
+m4_include([build/ax_require_defined.m4])
```

Comment:
This can be sorted alphabetically. Autoconf includes these at the beginning anyway regardless when calling some macro.